### PR TITLE
Added h2 configuration for book service and order service, upgraded java to 11

### PR DESCRIPTION
**Pull request by sreekanthvaripalli**
1.Minor refactoring for entity
2.updated java version to 11
3.Added h2 configuration for book service

Screenshot 2021-01-17 at 5 31 02 PM
Screenshot 2021-01-17 at 5 30 21 PM

### CODE REVIEW

File: book-service/src/main/resources/application-local.properties [hunk 1]
```diff
@@ -0,0 +1,22 @@
+springfox.documentation.swagger.v2.path=/api-docs
+server.port=8080
+spring.jackson.date-format=com.anitechcs.bookservice.RFC3339DateFormat
+spring.jackson.serialization.WRITE_DATES_AS_TIMESTAMPS=false
+logging.level.io.swagger.models.parameters.AbstractSerializableParameter=error
+
+#H2 db configuration
+spring.datasource.url=jdbc:h2:mem:testdb
```
Comment (tapas4java) [inline book-service/src/main/resources/application-local.properties:8]: Can we use the same DB name as Postgres?

File: order-service/src/main/resources/application-local.properties [hunk 2]
```diff
@@ -0,0 +1,23 @@
+springfox.documentation.swagger.v2.path=/api-docs
+server.port=8080
+spring.jackson.date-format=com.anitechcs.orderservice.RFC3339DateFormat
+spring.jackson.serialization.WRITE_DATES_AS_TIMESTAMPS=false
+logging.level.io.swagger.models.parameters.AbstractSerializableParameter=error
+spring.jpa.properties.hibernate.globally_quoted_identifiers=true
+
+#H2 db configuration
+spring.datasource.url=jdbc:h2:mem:testdb
```
Comment (tapas4java) [inline order-service/src/main/resources/application-local.properties:9]: Can we use the same DB name as Postgres?

**Review by tapas4java — DISMISSED** [review]
Looks good to me. Just change the DB name if you think it make sense otherwise leave it.

File: order-service/src/main/resources/application-local.properties (hunk 2 above)
Comment (sreekanthvaripalli) [reply order-service/src/main/resources/application-local.properties:9]: Okay. Will update. But this is just a file name where our values going to saved in file format

File: book-service/src/main/resources/application-local.properties (hunk 1 above)
Comment (sreekanthvaripalli) [reply book-service/src/main/resources/application-local.properties]: done

Comment (sreekanthvaripalli) [reply book-service/src/main/resources/application-local.properties]: do i need to rebase with master or will be done automatically?

**Review by tapas4java — APPROVED** [review]
Approved!

**Pull request merged**